### PR TITLE
Dialog Dropdown alignment fixes

### PR DESCRIPTION
**Pull request by placegraphichere**
- fix for #2656 
- fixed blendmodes for group, shape, text
- fixed typeface, weight
- fixed effects blendmode
- fixed export scale, format
- fixed library dialog

- typeface dialog width

### CODE REVIEW

File: src/style/shared/dialog.less [hunk 1]
```diff
@@ -109,9 +109,27 @@ dialog::backdrop {
...
 }
 
+.column-28 + .dialog-blendmodes{
+    width: 29.2rem;
+}
+
+.column-14 + .dialog-blendmodes{
+    width: 15.6rem;
+    left: 5.1rem;
+    margin-top: -1px;
```
Comment (iwehrman) [inline src/style/shared/dialog.less]: Do you think this will work at all resolutions?

**Comment by iwehrman** [comment]
One more in Export scale:
image

**Comment by iwehrman** [comment]
One bug, one question. Back to @placegraphichere.

**Comment by iwehrman** [comment]
Also, a process suggestion: when you're actively working on fixing a bug, it's nice to set the "Fix In Progress" label on that bug. It helps a) make sure nobody else starts working on it without talking to you, and b) helps us remember to verify that the bug is closed when you're done. Also, whoever merges your changes should go back and swap the label to "Fixed But Not Closed" and re-assign it to either QE or the original filer to verify the fix.

**Comment by placegraphichere** [comment]
back to you @iwehrman

**Comment by iwehrman** [comment]
Very nice.

**Pull request merged**